### PR TITLE
Add instructions to build ce-arq before run the tests

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Cloud Enablement Testsuite - ce-testsuite
 
+#### Before start
+Make sure you have the ce-arq dependencies available in your local maven repository. If you don't have it yet just clone the project and install it:
+
+```sh
+$ git clone https://github.com/jboss-openshift/ce-arq.git
+
+$ ce ce-arq
```
Comment (jwendell) [inline README.md]: cd

File: README.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Cloud Enablement Testsuite - ce-testsuite
 
+#### Before start
```
Comment (jwendell) [inline README.md]: before starting

**Comment by spolti** [comment]
updated. thx. :)

**Comment by alesj** [comment]
Merged.

**Pull request closed without merging**